src/api/spec/features/webui/image_templates_spec.rb

```diff
@@ -1,32 +1,56 @@
 require "browser_helper"
+require 'feature/testing'
```

Comment:
Can you please move this to `spec/support/features` and generally include it in `rails_helper`? Or is this very expensive?